web/src/lib/components/admin-page/settings/ffmpeg/ffmpeg-settings.svelte

```diff
@@ -65,212 +65,237 @@
+              name="videoContainers"
+              options={[
+                { value: VideoContainer.Mov, text: 'MOV' },
+                { value: VideoContainer.Ogg, text: 'Ogg' },
+                { value: VideoContainer.Webm, text: 'WebM' },
+              ]}
+              isEdited={!isEqual(
+                sortBy(config.ffmpeg.acceptedContainers),
+                sortBy(savedConfig.ffmpeg.acceptedContainers),
+              )}
+            />
+          </div>
+        </SettingAccordion>
 
-        <SettingCheckboxes
-          label={$t('admin.transcoding_accepted_video_codecs')}
-          {disabled}
-          desc={$t('admin.transcoding_accepted_video_codecs_description')}
-          bind:value={config.ffmpeg.acceptedVideoCodecs}
-          name="videoCodecs"
-          options={[
-            { value: VideoCodec.H264, text: 'H.264' },
-            { value: VideoCodec.Hevc, text: 'HEVC' },
-            { value: VideoCodec.Vp9, text: 'VP9' },
-            { value: VideoCodec.Av1, text: 'AV1' },
-          ]}
-          isEdited={!isEqual(sortBy(config.ffmpeg.acceptedVideoCodecs), sortBy(savedConfig.ffmpeg.acceptedVideoCodecs))}
-        />
+        <SettingAccordion
+          key="encoding-options"
+          title={$t('admin.transcoding_encoding_options')}
+          subtitle={$t('admin.transcoding_encoding_options_description')}
+        >
+          <div class="ml-4 mt-4 flex flex-col gap-4">
+            <SettingSelect
+              label={$t('admin.transcoding_video_codec')}
+              {disabled}
+              desc={$t('admin.transcoding_video_codec_description')}
+              bind:value={config.ffmpeg.targetVideoCodec}
+              options={[
+                { value: VideoCodec.H264, text: 'h264' },
+                { value: VideoCodec.Hevc, text: 'hevc' },
+                { value: VideoCodec.Vp9, text: 'vp9' },
+                { value: VideoCodec.Av1, text: 'av1' },
+              ]}
+              name="vcodec"
+              isEdited={config.ffmpeg.targetVideoCodec !== savedConfig.ffmpeg.targetVideoCodec}
+              onSelect={() => (config.ffmpeg.acceptedVideoCodecs = [config.ffmpeg.targetVideoCodec])}
+            />
 
-        <SettingCheckboxes
-          label={$t('admin.transcoding_accepted_audio_codecs')}
-          {disabled}
-          desc={$t('admin.transcoding_accepted_audio_codecs_description')}
-          bind:value={config.ffmpeg.acceptedAudioCodecs}
-          name="audioCodecs"
-          options={[
-            { value: AudioCodec.Aac, text: 'AAC' },
-            { value: AudioCodec.Mp3, text: 'MP3' },
-            { value: AudioCodec.Libopus, text: 'Opus' },
-            { value: AudioCodec.PcmS16Le, text: 'PCM (16 bit)' },
-          ]}
-          isEdited={!isEqual(sortBy(config.ffmpeg.acceptedAudioCodecs), sortBy(savedConfig.ffmpeg.acceptedAudioCodecs))}
-        />
+            <!-- PCM is excluded here since it's a bad choice for users storage-wise -->
```

Comment:
I think we should not exclude it as if the users need it; they should be able to choose the option. What do you think @mertalev

Reply:
PCM is just a bad option. Maybe we can offer FLAC in the future if there's a demand for lossless audio, but PCM is much bigger than even FLAC.